docs/features/shipping-javadoc.md

```diff
@@ -0,0 +1,69 @@
+### Shipping Javadoc
+
+See also "[How Shipkit Works](/docs/how-shipkit-works.md)" documentation index.
+Please help us with docs and submit a PR with improvements!
+
+Your users need to know documentation of your project. You want release documentation? 
+Write it as Javadoc and publish to separate (or the same) repo. Make it available for everyone. 
+
+#### Intro
+
+Gradle build generate Jar with Javadoc by default. Shipkit allows you to publish your Javadoc in GitHub repository.
```

Comment:
I'd mention that we are pushing "html" to the repo/brach so that GitHub pages show browsable, readable html.